test/system/001-version.bats

```diff
@@ -20,6 +20,7 @@ load 'libs/bats-assert/load'
 load 'libs/helpers.bash'
 
 setup() {
+  bats_require_minimum_version 1.5.0
```

Comment:
Note that `bats_require_minimum_version` itself is only available from Bats 1.7.0.  [Here's](https://github.com/containers/toolbox/commit/e22a82fec8e59c5a0f38f1ec6dc6899593842bfd) a similar commit with more details.

So, I suggest requiring 1.7.0.

Reply:
I took the liberty to change this and split it out in to https://github.com/containers/toolbox/pull/1315 because it will be really good to fix the warnings.  :)